frontend/packages/dev-console/src/components/project-access/ProjectAccessPage.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { match as RMatch } from 'react-router';
+import { Firehose } from '@console/internal/components/utils';
+import { NamespaceBar } from '@console/internal/components/namespace';
+import DefaultPage from '../DefaultPage';
+import ProjectAccessForm, { ProjectAccessFormProps } from './ProjectAccessForm';
+import ProjectAccess from './ProjectAccess';
+
+export interface ProjectAccessPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+}
+
+const ProjectAccessPage: React.FC<ProjectAccessPageProps> = ({ match }) => {
+  const namespace = match.params.ns;
+  const props: ProjectAccessFormProps = {
+    ProjectAccessFormComponent: ProjectAccess,
+    formName: 'project access',
+    namespace,
+  };
+  return (
+    <React.Fragment>
+      <NamespaceBar />
+      {namespace ? (
+        <React.Fragment>
+          <Firehose
+            resources={[
+              { namespace, kind: 'RoleBinding', prop: 'roleBindings', isList: true },
+              { kind: 'Project', prop: 'projects', isList: true },
```

Comment:
Why fetch projects? You are displaying role bindings only.

Reply:
I did this because the role Binding request to fetch all the roleBindings, when no projects exist, fails with response as forbidden and the value of `roleBindings.loaded` never becomes `true`. I thought if I simply check for `roleBindings.loaded` it would be difficult to differentiate between scenarios like `when no projects exist` & `user not having permission to list roleBindings`

Reply:
Why would we fetch rolebindings when no project exists? we are supposed to be scoped to a specific namespace and therefore shouldn't load the form unless in a project scope.